processors/blockprocessor/block_processor.go

```diff
@@ -75,14 +181,14 @@ func addGenesisBlock(l *ledger.Ledger, handler func(block *ledgercore.ValidatedB
 	return nil
 }
 
-// Process a raw algod block
-func (proc *blockProcessor) Process(blockCert *rpcs.EncodedBlockCert) error {
+// ProcessBlockCert Process a raw algod block
+func (proc *blockProcessor) ProcessBlockCert(blockCert *rpcs.EncodedBlockCert) error {
```

Comment:
ProcessBlockCert() and Proccess() seem to have a lot of overlap. why do we need both?